apps/backend/src/db/migrations/20240822143738_add_manage_user_access_to_permission.ts

```diff
@@ -0,0 +1,24 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "permissions";
+
+const PermissionName = {
+	MANAGE_USER_ACCESS: "Manage User Access",
+};
```

Comment:
Use permission key instead

Reply:
Still not fixed

Reply:
This is why I am confused... I've already done it

```
function down(knex: Knex): Promise<void> {
	return knex(TABLE_NAME)
		.where({
			key: PermissionKey.MANAGE_USER_ACCESS,
		})
		.del();
}
```

Reply:
oh, didn't see it, then ok